openmetadata-docs/content/deployment/docker/index.md

```diff
@@ -4,39 +4,174 @@ slug: /deployment/docker
+```
+
+In a few seconds, you should be able to access the OpenMetadata UI at [http://localhost:8585](http://localhost:8585)
+## Port Mapping / Port Forwarding 
 
 We are shipping the OpenMetadata server and UI at `8585`, and the ingestion container (Airflow) at `8080`. You can
-take a look at the official Docker [docs](https://docs.docker.com/compose/compose-file/#ports). As an example, You could
-update the ports to serve Airflow at `1234` with:
+change the port number's according to your requirement. As an example, You could
+update the ports to serve OpenMetadata Server and UI  at port `80`
+
+To achieve this 
+- You just have to update the ports mapping of the openmetadata-server in the `docker-compose.yml` file
 
 ```yaml
 ports:
-  - "1234:8080"
+  - "80:8080"
+```
+- Once the port is updated if there are any containers running remove them first using `docker-compose down` command and then  recreate the containers once again by below command 
+```commandline
+docker compose up --build -d 
+```
+
+## PROD Deployment of OpenMetadata Using Docker
+If you are planning on going to PROD, we recommend to validate below points:
+- MySQL and OpenSearch (ElasticSearch) are available.
+- OpenMetadata-Server require the minimum configuration of 2vCPU and 6Memory (GiB)
+- OpenMetadata-Ingestion require the minimum configuration of 2vCPU and 8Memory (GiB)
+- We also recommend to bind Docker Volumes for data persistence. Learn how to do so [here](/deployment/docker/volumes).
+### Steps for Deploying Ingestion 
+- Download the docker-compose.yml file from the release page [here](https://github.com/open-metadata/OpenMetadata/releases).
+- Update the environment variables below for OpenMetadata-Ingestion Docker Compose backed systems to connect with Database. 
+```
+# MySQL Environment Variables for ingestion service
+DB_HOST: '<YOUR_RDS_HOST_NAME>'
+DB_PORT: '<YOUR_RDS_PORT>'
+AIRFLOW_DB: '<YOUR_RDS_AIRFLOW_DB>'
+AIRFLOW_DB_SCHEME: '<YOUR_RDS_AIRFLOW_DB_SCHEME>'
+DB_USER: '<YOUR_RDS_AIRFLOW_DB_USER>'
+DB_PASSWORD: '<YOUR_RDS_AIRFLOW_DB_PASSWORD>'
+```
+Once the environment variables values with the RDS are updated then provide this environment variable file as part of docker compose command.
+
 ```
+docker compose --env-file ./config/.env.prod up -d openmetadata_ingestion
+```
+### Steps for Deploying OpenMetadata-Server
+- Download the docker-compose.yml file from the release page [here](https://github.com/open-metadata/OpenMetadata/releases).
+- Update the environment variables below for OpenMetadata-Ingestion Docker Compose backed systems to connect with Database and ElasticSearch and Ingestion.
+```
+# MySQL Environment Variables
+DB_DRIVER_CLASS='com.mysql.cj.jdbc.Driver'
+DB_SCHEME='mysql'
+DB_USE_SSL='true'
+MYSQL_USER_PASSWORD='<YOUR_RDS_USER_PASSWORD>'
+DB_SCHEME='mysql'
+MYSQL_HOST='<YOUR_RDS_HOST_NAME>'
+MYSQL_USER='<YOUR_RDS_USER_NAME>'
+MYSQL_DATABASE='<YOUR_RDS_DATABASE_NAME>'
+MYSQL_PORT='<YOUR_RDS_PORT>'
```

Comment:
Update the ENV variable names to latest ones that starts with DB_*.

Update the placeholder to be more generic!